packages/ui-kit/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,106 @@
+import { FC, ReactNode, useState } from "react";
+import clsx from "clsx";
+import { Combobox } from "@headlessui/react";
+import { nanoid } from "nanoid";
+
+import styles from "./Select.module.css";
+import { ChevronDownIcon } from "../icons";
+
+interface Option {
+  label: string | ReactNode;
+  before?: string | ReactNode;
+  id?: string;
+  [key: string]: unknown;
+}
+
+export interface SelectProps {
+  options: Option[];
+  selected: Option;
+  error?: boolean;
+  disabled?: boolean;
+  className?: {
+    container?: string;
+    triggerBefore?: string;
+    trigger?: string;
+    triggerArrow?: string;
+    options?: string;
+    optionBefore?: string;
+    option?: string;
+  };
+  onChange?: (option: Option) => void;
+}
+
+export const Select: FC<SelectProps> = ({
```

Comment:
(storybook ->) Currently select adjusts its width to its selected element, and I don't think it should. Same goes for disabled (altho I'd talk to Paweł about keeping some form of the vertical divider and chevron cause the empty space looks weird)

Reply:
Container should be always 100% width.